content/page/who.en.md

```diff
@@ -12,7 +12,7 @@ Exodus Privacy is a French (1901 law) non-profit organization. This organization
 
 <div class="row justify-content-md-center">
     <div class="col-md-3 mt-3 text-center">
-        <i class="far fa-3x fa-user mt-2 ml-auto mr-auto text-primary"></i>
+	<img class="w-25 rounded-circle" src="/media/avatar/capslock.jpg">
```

Comment:
Indentation seems weird, seems to be 1 tab instead of 8 spaces.
I only made the suggestion on this line but the change should be made on every line.

```suggestion
        <img class="w-25 rounded-circle" src="/media/avatar/capslock.jpg">
```